recipes/pyppeteer/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pyppeteer" %}
+{% set version = "0.0.25" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 51fe769b722a1718043b74d12c20420f29e0dd9eeea2b66652b7f93a9ad465dd
+  patches:
+    - setup.patch
+
+build:
+  noarch: python
```

Comment:
`noarch: python` should be used for Python 2 and 3 compatible libraries

Reply:
The noarch wiki page says othewise. Noarch is about win,  macos, linux builds. The fact that it can unify python versions too is just a bonus.

Reply:
https://conda-forge.org/docs/meta.html#building-noarch-packages